server/protocols/webadmin/webadmin-jmap/src/main/java/org/apache/james/webadmin/data/jmap/dto/UserIdentity.java

```diff
@@ -113,6 +131,43 @@ public UserIdentity(String name, String email, String id,
         this.replyTo = replyTo;
     }
 
+    @JsonCreator
+    public UserIdentity(@JsonProperty("name") String name,
```

Comment:
> why there is no mayDelete field 

The current API does not accept input `mayDelete` field

Quote:
```
org.apache.james.jmap.api.identity.IdentityRepository#save
mayDelete alway is TRUE
```

Should we modify it for web-admin case?

> I would see more this as a UserIdentityCreate or something similar

Yes, it will be clear if we have a separate object `UserIdentityCreate` / `UserIdentiteUpdate`,
But, IMO, it is simple pojo, we can re-use it in a single object `UserIdentity`, less boilerplate. 
So, I didn't create UserIdentityCreate...

Reply:
Forget the mayDelete field remark, as said in latest comment it's indeed server-set.

> Yes, it will be clear if we have a separate object UserIdentityCreate / UserIdentiteUpdate,
> But, IMO, it is simple pojo, we can re-use it in a single object UserIdentity, less boilerplate.

Honestly it was a bit confusing to me, I had to re-read it to understand what it was for^^'